docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js

```diff
@@ -3,37 +3,58 @@
  * Sets app config for initialization.
  */
 
-import initialize from '@nciocpl/sitewide-search-app';
-import { getCanonicalURL, getDocumentLanguage, getMetaData, parseUrl } from 'Utilities';
+// #3003 remove glossary app module now that generic app modules exist
+// import initialize from '@nciocpl/sitewide-search-app';
+const initialize = () => {
+  throw new Error("Replaced via #3003. This is now a generic app module");
+};
 
-const instanceConfig = window.sitewideSearchAppSettings;
+import {
+  getCanonicalURL,
+  getDocumentLanguage,
+  getMetaData,
+  parseUrl
+} from "Utilities";
 
-const metaTags = getMetaData([
-  ['name', 'dcterms.subject'],
-  ['name', 'dcterms.isPartOf'],
-  ['name', 'dcterms.issued'],
-  ['property', 'og:site_name'],
-  ['property', 'og:title']
-], document);
+const instanceConfig = window.sitewideSearchAppSettings;
 
+const metaTags = getMetaData(
+  [
+    ['name', 'dcterms.subject'],
+    ['name', 'dcterms.isPartOf'],
+    ['name', 'dcterms.issued'],
+    ['property', 'og:site_name'],
+    ['property', 'og:title']
+  ],
+  document
+);
```

Comment:
I'm not sure what I'm seeing here.  This file should be going away once the Drupal module is completed, and yet, there are changes being made.

That shouldn't affect anything if the code never executes (and the changes to initialize method should prevent that), but I'm confused.  Some of this is just reformatting, but there's new code as well (e.g. the various Utilities imports).

Reply:
No it is all just code reformatting here.  The utilities imports was broken out on multiple lines.

Reply:
Oh, I see. Dang that's confusing! 😕